ray-operator/config/openshift/params.yaml

```diff
@@ -0,0 +1,7 @@
+varReference:
+  - path: subjects[]/namespace
+    kind: ClusterRoleBinding
```

Comment:
I don't think it's need anymore as my understanding is it was required for the binding to the edit ClusterRole which you removed.

Reply:
Thanks @astefanutti :)